app/models/job.rb

```diff
@@ -61,7 +61,7 @@ def duration
   end
 
   def can_be_cancelled_by?(user)
-    started_by?(user) || user.admin? || user.admin_for?(project)
+    started_by?(user) || user.admin? || user.deployer_for?(project)
```

Comment:
I think this can just be `user.deployer_for?(project)` since only deployers can start a deploy ...
@ragurney sounds alright ?

Reply:
I changed it to `user.admin? || user.deployer_for?(project)` so that all admins and all deployers on the project will have permission to cancel the deploy.